### DISCUSSION TITLE
How to escape emoji / 絵文字のエスケープ方法

### DISCUSSION THREAD

**Discussion by MinaiBytes** [growilabs/growi#6578] [Help / Questions] [answered]
例えば、以下のように入力した場合
```
C:\Windows\～
```

Growiでは以下のように表示されます
![image](https://user-images.githubusercontent.com/56664277/190036079-f859b7bd-dd14-4bbd-b160-0cf245d6e122.png)

どうすれば、絵文字を表示せずに済みますでしょうか？

**Comment by yuki-takei** [comment, marked as answer]
別ルートで同様のお問合せを複数いただいておりますが、現状回避策がない状況です。

GROWI で利用している markdown-it というライブラリか、またはそれを利用している GROWI のコードが原因である可能性が高いのですが、詳しくは調査できていません。

参考: https://github.com/markdown-it/markdown-it-emoji/blob/master/lib/data/shortcuts.js

というのも、実はあと 1～2 ヶ月後には GROWI v6 が出るんですが、そちらは markdown 解析に markdown-it ではなく remark というものを使うようになります。remark では同様の問題は起きないことを確認しています。

そのため今この問題を解決するために markdown-it (および GROWI の markdown-it 利用版のソース)について挙動を深く調査していく意味は薄いと思っていて、申し訳ないですが v6 での解決をお待ちいただきたいという感じです。

**Comment by yuki-takei** [comment]
v6 以降では再現しないためクローズします
